pkg/controllers/provisioning/binpacking/packer.go

```diff
@@ -93,17 +93,19 @@ func (p *Packer) Pack(ctx context.Context, schedule *scheduling.Schedule, instan
 			continue
 		}
 		key, err := hashstructure.Hash(packing, hashstructure.FormatV2, &hashstructure.HashOptions{SlicesAsSets: true})
-		if err == nil {
-			if mainPack, ok := packs[key]; ok {
-				mainPack.NodeQuantity++
-				mainPack.Pods = append(mainPack.Pods, packing.Pods...)
-				continue
-			} else {
-				packs[key] = packing
-			}
+		if err != nil {
+			logging.FromContext(ctx).Fatal("Unable to hash packings while binpacking")
```

Comment:
Can you include the error in the log line so we can reassess our assumptions if this every happens.